src/matrixroomhandler.ts

```diff
@@ -83,14 +86,28 @@ export class MatrixRoomHandler {
         };
     }
 
-    public setBridge(bridge: Bridge) {
+    public setBridge(bridge: Bridge, roomStore: DbRoomStore) {
         this.bridge = bridge;
+        this.roomStore = roomStore;
     }
 
     public async OnAliasQueried(alias: string, roomId: string) {
         log.verbose(`Got OnAliasQueried for ${alias} ${roomId}`);
         let channel: Discord.GuildChannel;
         try {
+            // We previously stored the room as an alias.
```

Comment:
why not migrate this in the migration?

Reply:
huh? No, we have to store the matrix_id as an alias in `createMatrixRoom` because we don't know the roomId yet.